src/o365/todo/commands/list/list-remove.ts

```diff
@@ -0,0 +1,159 @@
+}
+
+class TodoListRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.LIST_REMOVE}`;
+  }
+
+  public get description(): string {
+    return 'Removes a Microsoft To Do task list';
+  }
+
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+
+    const getListId = () => {
+      if (args.options.name) {
+        // Search list by its name
+        const requestOptions: any = {
+          url: `${this.resource}/beta/me/todo/lists?$filter=displayName eq '${escape(args.options.name)}'`,
+          headers: {
+            accept: "application/json;odata.metadata=none"
+          },
+          json: true
+        };
+        return request.get(requestOptions)
+          .then((response: any) => response.value && response.value.length === 1 ? response.value[0].id : null);
+      }
+
+      return Promise.resolve(args.options.id);
+    };
+
+    const removeList = () => {
+      getListId().then(listId => {
+        if (!listId) {
+          cb(new CommandError(`The list ${args.options.name} cannot be found`));
+          return;
+        }
+
+        const requestOptions: any = {
+          url: `${this.resource}/beta/me/todo/lists/${listId}`,
+          headers: {
+            accept: "application/json;odata.metadata=none"
+          },
+          json: true
+        };
+
+        request
+          .delete(requestOptions)
+          .then((): void => {
+
+            if (this.verbose) {
+              cmd.log(vorpal.chalk.green('DONE'));
+            }
+
+            cb();
+          }, (err: any) => this.handleRejectedODataJsonPromise(err, cmd, cb));
+      }, (err: any) => this.handleRejectedODataJsonPromise(err, cmd, cb));
+
+    };
+
+    if (args.options.confirm) {
+      removeList();
+    }
+    else {
+      cmd.prompt(
+        {
+          type: "confirm",
+          name: "continue",
+          default: false,
+          message: `Are you sure you want to remove the list ${args.options.id || args.options.name}?`
+        },
+        (result: { continue: boolean }): void => {
+          if (!result.continue) {
+            cb();
+          }
+          else {
+            removeList();
+          }
+        }
+      );
+    }
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.name = typeof args.options.name !== 'undefined';
+    telemetryProps.id = typeof args.options.id !== 'undefined';
+    telemetryProps.confirm = args.options.confirm;
+    return telemetryProps;
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-n, --name [name]',
+        description: `The name of the task list to remove. Specify either id or name but not both`
+      },
+      {
+        option: '-i, --id [id]',
+        description: `The ID of the list to remove. Specify either id or name but not both`
```

Comment:
Let's rename this to `The ID of the task list to remove. Specify either id or name but not both`